pkg/components/gtime/gtime.go

```diff
@@ -22,16 +22,18 @@ func ParseInterval(inp string) (time.Duration, error) {
 
 	num := int(dur)
 
-	now := time.Now()
+	// Use UTC to ensure that the interval is deterministic, and daylight saving
+	// doesn't cause surprises
+	now := time.Now().UTC()
 	switch period {
 	case "d":
-		return now.Sub(now.AddDate(0, 0, -num)), nil
+		return now.AddDate(0, 0, num).Sub(now), nil
```

Comment:
To get rid of relative to the current wall time, can we return `dur * 24 * time.Hour` and so on?

Reply:
@marefr this function is meant to be relative to wall time, if we want fixed durations, `ParseDuration` should be used instead. I'm actually wondering if we can use fixed durations everywhere? Would certainly make things easier.